docs/contributing-guide/6. submitting-a-pr.md

```diff
@@ -61,7 +61,7 @@ If you are making an ui change in your PR, you might invalidate some screen snap
 1. Check the failing github actions to see if that is a desired UI change
 
    ![screenshot showing downloadable snapshot artifacts in a failed github action](../images/check-snapshots.png)
-1. Add `update_snapshot` label to the PR
+1. Add `update_snapshot` label to the PR - Note: Add this label after you create your PR. Github actions bot will not detect the label if added while creating the PR. If action bot does not take action on the label remove it and add the label again, this should start the update process.
```

Comment:
Dropping comment here, I would recommend a fix for this instead of documenting the flaw.
We should be able to add something like
```
on:
  pull_request
    types: [opened, reopened]
```
to the update snapshots workflow file

See more: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

Reply:
i agree with James's comment. Can we do that instead?